src/napari_topostats/_widget.py

```diff
@@ -240,276 +202,3 @@ def get_functions(self):
                 elif callable(func):
                     functions[display_name] = magicgui(func)
         return functions
-
-
-def remove_scars_from_image(  # pylint: disable=too-many-positional-arguments
```

Comment:
I thought this was being removed because the scar removal from `TopoStats.Filters` was being removed but I see in `utils.py` that these are no longer imported either.

Haven't time to test the GUI but does the ability to Filter images still remain?